fda/grid.py

```diff
@@ -278,6 +282,35 @@ def shape(self):
         """
         return self.data_matrix.shape
 
+    @property
+    def _evaluator(self):
+
+        if self._interpolator_evaluator is not None:
+            return self._interpolator_evaluator
+
+        if self.interpolator is None:
+            # By default a Linear Spline Interpolator is used
+            self.interpolator = GridSplineInterpolator()
+
+        self._interpolator_evaluator = self.interpolator._construct_interpolator(self)
+
+        return self._interpolator_evaluator
+
+
+    def set_interpolation(self, interpolator):
+
+        self.interpolator = interpolator
```

Comment:
The evaluator must change too. In fact, why we have a separate "evaluator". Couldn't the FDataGrid get passed to the interpolator when evaluating?

Reply:
Bug fixed in ef5ce83.
The reason to have the evaluator is to avoid construct the scipy interpolators every time evaluate is called.